README.md

```diff
@@ -33,9 +33,8 @@ your model history to change its parameters.
  3D models to create high quality production-ready drawings.
 
 * **Designed for your needs** FreeCAD is designed to fit a wide range of uses
- including product design, mechanical engineering and architecture. 
- Whether you are a hobbyist, programmer, experienced CAD user, student or teacher,
- you will hopefully feel right at home.
+including product design, mechanical engineering and architecture,
+whether you are a hobbyist, programmer, experienced CAD user, student or teacher,
```

Comment:
We're almost there, lol (thx for your patience) but you need to swap the comma for a period after `teacher`  
regex: `s/teacher,$/teacher\./`

Edit: I just sent a PR to your branch :smile:

Reply:
Just when I had surrendered to my faith of being here forever. :)